src/main/webapp/createEvent.html

```diff
@@ -2,16 +2,23 @@
 <html>
     <head>
         <meta charset="UTF-8">
-        <title>Create Event</title>
+        <title>The Spot - Create Events</title>
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css">
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
```

Comment:
You have version 3.5.1 below, so idk that you need this one as well

Reply:
just tested it out and I needed the 3.4.1, not the 3.5.1